src/components/TodoListHeader.js

```diff
@@ -1,27 +1,39 @@
 import React from "react";
 import "./TodoListHeader.css";
+import TextInput from "./TextInput";
 
 class TodoListHeader extends React.Component {
```

Comment:
Check if this component is rendering when you change a caption of an item or check it as completed. Should it rerender on every change?